src/components/visualizations/chart/test/chartOptionsBuilder.spec.ts

```diff
@@ -1561,6 +1560,47 @@ describe("chartOptionsBuilder", () => {
                 });
             });
         });
+
+        describe("in use case of heatmap", () => {
+            const dataSet = fixtures.heatmapEmptyCells;
+            const { measureGroup } = getMVS(dataSet);
+            const executionResultData: Execution.DataValue[][] = dataSet.executionResult.data;
+            const heatmapSeries = getHeatmapSeries(executionResultData, measureGroup);
+            const heatmapDataPoints = heatmapSeries[0].data;
+            const firstEmptyCellIndex = heatmapDataPoints.findIndex(point => isNil(point.value));
+
+            it("should return only one serie", () => {
+                expect(heatmapSeries.length).toBe(1);
+            });
+
+            it("should have two data point at null value", () => {
+                const nullDataCount = executionResultData.reduce(
+                    (result, data) => {
+                        result.count += data.filter(isNil).length;
+                        return result;
+                    },
+                    { count: 0 },
+                ).count;
+                const nullPointCount = heatmapSeries[0].data.map(data => data.value).filter(isNil).length;
+                expect(nullPointCount).toBe(nullDataCount * 2);
+            });
+
+            it("First empty point should have gray border", () => {
+                const firstEmptyCell = heatmapDataPoints[firstEmptyCellIndex];
+                expect(firstEmptyCell.borderColor).toEqual(GRAY);
+                expect(firstEmptyCell.borderWidth).toBe(1);
+                expect(firstEmptyCell.pointPadding).toBe(undefined);
+                expect(firstEmptyCell.color).toBe("transparent");
+            });
+
+            it("Second empty point should have stripes inside", () => {
```

Comment:
`Second empty point should have stripes inside` -> `should second empty point have stripes inside` please